Upcoming release/c8097ed3-15e7-4941-8eb1-7bed1fadef9d.md

```diff
@@ -56,6 +56,7 @@ To search by products, send the request:
 | `GET https://glue.mysprykershop.com/catalog-search?q=Sony&sort=name_desc` | Set sorting order descending.   |
 | `GET https://glue.mysprykershop.com/catalog-search?q=Sony&sort=rating` | Sort found products by rating.   |
 | `GET https://glue.mysprykershop.com/catalog-search?q=Sony&sort=price_asc` | Sort found products by price ascending.   |
+| `GET https://glue.mysprykershop.com/catalog-search?q=Sony&sort=popularity` | Sort found products by popularity.   |
```

Comment:
please revert all changes from this file, because it's not a part of 202108.0 release, it was released after code freeze